pkg/shell/templates/change-port.html

```diff
@@ -0,0 +1,47 @@
+<div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal"
+        aria-hidden="true">&times;</button>
+    <h4 class="modal-title" translatable="yes">Could not contact {{host}}</h4>
+</div>
+
+<div class="modal-body">
+    <div translatable="yes">
```

Comment:
I don't think we can use this with a translatable attribute mixed with mustache elements.